src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/testhosts/CustomTextMessageBindingElement.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Reviewers: these additional files were copied directly from the old self-hosted server's TestResources/CustomBinding folder, and the namespace was made consistent with the other files in this folder.  Regarding whether or not they should live under a separate folder should be discussed for a later PR that makes the App_code file hierarchy less flattened.